test/tests/functional/pbs_maintenance_reservations.py

```diff
@@ -440,36 +440,53 @@ def test_maintenance_and_reservation_not_overlap2(self):
                     'reserve_substate': 2}
         self.server.expect(RESV, exp_attr, id=rid1)
 
+    def get_reg_expr(self, _mom, _host):
+        """
+        This function forms the match string based on cpuset mom
+        """
+        if _mom.is_cpuset_mom():
+            n = self.server.status(NODE)
+            cpuset_nodes = [i['id'] for i in n if i['Mom'] == _mom.hostname]
+            reg_str = []
+            if (len(cpuset_nodes) - 1) > 1:
+                for i in range(0, len(cpuset_nodes)-1):
+                    host = _host + '\[%s\]' % i
+                    match_str = '\(%s:ncpus=[0-9]+\)' % host
+                    reg_str.append(match_str)
+                reg_str = ('\+').join(reg_str)
```

Comment:
Can you split reg_str into 2 variables?  Right now it's called a string, but is first a list, then a string.  Can you make the list a different variable and then assign it into reg_str?

Reply:
I have modified entire list->string. I have considered reg_str as a string and concatenated string to it.